exporters/prometheus/build.gradle.kts

```diff
@@ -12,14 +12,15 @@ dependencies {
   api(project(":sdk:metrics"))
 
   implementation(project(":sdk-extensions:autoconfigure-spi"))
+  implementation("io.prometheus:prometheus-metrics-exporter-httpserver")
```

Comment:
What kind of compatibility guarantees does `prometheus-metrics-exporter-httpserver` offer? Couldn't find anything obvious on the [github repo](https://github.com/prometheus/client_java?tab=readme-ov-file) or [docs site](https://prometheus.github.io/client_java/).

Reply:
Good point, we should document this. I'll put it on my TODO list. All `1.x` versions should be compatible, if there is a breaking change we should release `2.x`. I also think that the API is small enough that we won't need a `2.x` in the foreseeable future.